frontend/src/js/route-config/route-config.js

```diff
@@ -362,10 +362,47 @@
             name: "web.profile",
             parent: "web",
             url: "/profile",
-            templateUrl: baseUrl + "/web/profile.html",
+            templateUrl: baseUrl + "/web/profile/profile.html",
             title: "Profile",
             controller: 'profileCtrl',
             controllerAs: 'profile',
+            redirectTo: "web.profile.Updateprofile",
+            authenticate: true
+        };
+
+        var AuthToken = {
+            name: "web.profile.AuthToken",
+            parent: "web.profile",
+            url: "/auth-token",
+            templateUrl: baseUrl + "/web/auth/get-token.html",
+            title: 'AuthToken',
+            authenticate: true
+        };
+
+        var Updateprofile = {
+            name: "web.profile.Updateprofile",
+            parent: "web.profile",
+            url: "/update-profile",
+            templateUrl: baseUrl + "/web/profile/edit-profile/update-profile.html",
+            title: 'Updateprofile',
+            authenticate: true
+        };
+
+        var Editprofile = {
+            name: "web.profile.Editprofile",
+            parent: "web.profile",
+            url: "/edit-profile",
+            templateUrl: baseUrl + "/web/profile/edit-profile/edit-profile.html",
+            title: 'Editprofile',
+            authenticate: true
+        };
+
+        var ChangePassword = {
+            name: "web.profile.ChangePassword",
+            parent: "web.profile",
+            url: "/change-password",
+            templateUrl: baseUrl + "/web/change-password.html",
+            title: 'ChangePassword',
```

Comment:
Keep the variable names like `update_profile` as they were earlier. The title should be `Update Profile`. Don't introduce new formats unless needed.